<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #1a1a1a;
}
QScrollBar:horizontal, QScrollBar:vertical {
	background-color: #1a1a1a;
}
QScrollBar::sub-page:horizontal, QScrollBar::add-page:horizontal {
	background-color: #646464;
}
QMenuBar::item:pressed, QMenuBar::item:selected, QMenu::item:selected {
	background-color: #646464;
}
QMenuBar {
	border-bottom: 1px solid #646464;
}
QSlider::handle:horizontal {
	background-color: #646464;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="6">
     <widget class="QLabel" name="totalTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>--:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <spacer name="bottomVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="8">
     <widget class="QTreeView" name="libraryView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeView {
	background-color: #1a1a1a;
	alternate-background-color: #646464;
	color: white;
}
QTreeView::item:selected {
	background-color: rgb(9, 1, 122);
	color: white;
}
QHeaderView::section { 
	color: white;
    background-color: #1a1a1a;
	font-weight: bold;
}
QHeaderView::section:hover { 
    background-color: #646464;
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="timePassed">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>--:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="leftHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>115</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="currentTrackLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <spacer name="rightHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>115</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="PlayerControls" name="controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>67</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;
color: white;

QAction::item:selected {
	background-color: red;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddToLibrary"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="viewTrackId"/>
    <addaction name="viewArtist"/>
    <addaction name="viewAlbum"/>
    <addaction name="viewTrackNum"/>
    <addaction name="viewYear"/>
    <addaction name="viewGenre"/>
    <addaction name="viewDuration"/>
    <addaction name="viewLocation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionAddToLibrary">
   <property name="text">
    <string>Add File(s) to Library</string>
   </property>
  </action>
  <action name="viewTrackId">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track ID</string>
   </property>
  </action>
  <action name="viewArtist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </action>
  <action name="viewAlbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="viewTrackNum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Track Number</string>
   </property>
  </action>
  <action name="viewYear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </action>
  <action name="viewGenre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </action>
  <action name="viewDuration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </action>
  <action name="viewLocation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlayerControls</class>
   <extends>QWidget</extends>
   <header>playercontrols.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
